content/en/docs/components/pipelines/pipelines-quickstart.md

```diff
@@ -2,74 +2,38 @@
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
 
++++                 
 {{% stable-status %}}
 
-Use this guide if you want to get a simple pipeline running quickly in
-Kubeflow Pipelines.
-
-* This quickstart guide shows you how to use one of the samples that come with 
-  the Kubeflow Pipelines installation and are visible on the Kubeflow Pipelines
-  user interface (UI). You can use this guide as an introduction to the 
-  Kubeflow Pipelines UI.
-* The end-to-end tutorial shows you how to prepare and compile a pipeline, 
-  upload it to Kubeflow Pipelines, then run it.
-
-## Deploy Kubeflow and open the pipelines UI
-
-Follow these steps to deploy Kubeflow and open the pipelines dashboard:
+Use this guide if you want to get an introduction to the Kubeflow Piplines user interface (UI) and get a simple pipeline running quickly. 
 
-1. Follow the guide to [deploying Kubeflow on GCP](/docs/gke/deploy/).
+The goal with this quickstart guide is to shows how to use two of the samples that come with 
+the Kubeflow Pipelines installation and are visible on the Kubeflow Pipelines
+UI. You can use this guide as an introduction to the 
+Kubeflow Pipelines UI.
 
-    {{% pipelines-compatibility %}} 
+## Deploy Kubeflow and open the Kubeflow Pipelines UI
 
-1. When Kubeflow is running, access the Kubeflow UI at a URL of the form
-  `https://<deployment-name>.endpoints.<project>.cloud.goog/`, as described in the setup
-  guide. The Kubeflow UI looks like this:
-  <img src="/docs/images/central-ui.png" 
-    alt="Kubeflow UI"
-    class="mt-3 mb-3 border border-info rounded">
+There are several options to [deploy Kubeflow Pipelines](/docs/components/pipelines/installation/), follow the option that best suits your needs. If you are uncertain and just want to try out kubeflow pipelines it is recommended to start with the [standalone deployment](/docs/components/pipelines/installation/standalone-deployment/).
 
-    If you skipped the Cloud IAP option when deploying Kubeflow, or if you 
-    haven't yet set up your Kubeflow endpoint, you can access Kubeflow via 
-    `kubectl` and port-forwarding:
-    
-    1. Install `kubectl` if you haven't already done so, by running the 
-      following command on the command line: 
-      `gcloud components install kubectl`. For more information, see the 
-      [`kubectl` 
-      documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-
-    1. Run ```kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80``` and go to `http://localhost:8080/`.
-
-1. Click **Pipelines** to access the pipelines UI. The pipelines UI looks like
-  this:
-  <img src="/docs/images/pipelines-ui.png" 
-    alt="Pipelines UI"
-    class="mt-3 mb-3 border border-info rounded">
+Once you have deployed Kubeflow Pipelines make sure you can access the UI, this step depends on the method of deployment. 
```

Comment:
Suggested revision (grammar, clarity): 

Once you have deployed Kubeflow Pipelines, make sure you can access the UI. The steps to access the UI vary based on the method you used to deploy Kubeflow Pipelines.